kong/plugins/prometheus/api.lua

```diff
@@ -1,11 +1,11 @@
 local exporter = require "kong.plugins.prometheus.exporter"
 
-local printable_metric_data = function()
+local printable_metric_data = function(_)
   local buffer = {}
   -- override write_fn, since stream_api expect response to returned
   -- instead of ngx.print'ed
-  exporter.metric_data(function(data)
-    table.insert(buffer, table.concat(data, ""))
+  exporter.metric_data(function(new_metric_data)
+    table.insert(buffer, table.concat(new_metric_data, ""))
```

Comment:
localize `table.insert` and `table.concat`?

Reply:
I'm only trying to address the confusion related to the parameter.